src/index.ts

```diff
@@ -1,3 +1,7 @@
-export { default as expressMiddleware } from './middleware/index.js';
+export { default as expressGraphQLRateLimiter } from './middleware/index';
 
-export { default as rateLimiter } from './middleware/rateLimiterSetup.js';
+export { default as rateLimiter } from './middleware/rateLimiterSetup';
```

Comment:
Have you verified this works without file extensions?

Reply:
No but I was going too. I assumed the "add-js-extentsion" plugin would handle these just like all the others.

Reply:
It does not. The `.js` here is necessary. Somehow I think you knew that.